Reason: Fix double percent encoding of invites
Upstream: Taken from upstream commit https://github.com/Nheko-Reborn/nheko/commit/d94ac8 
--- src/ChatPage.cpp.orig
+++ src/ChatPage.cpp
@@ -1015,8 +1015,7 @@
 void
 ChatPage::joinRoom(const QString &room)
 {
-        // Percent escape the room ID
-        const auto room_id = QUrl::toPercentEncoding(room).toStdString();
+        const auto room_id = room.toStdString();
 
         http::client()->join_room(
           room_id, [this, room_id](const nlohmann::json &, mtx::http::RequestErr err) {
